app/views/business_follows/create.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>BusinessFollows#create</h1>
+<p>Find me in app/views/business_follows/create.html.erb</p>
```

Comment:
controller#create should not have associated view

Reply:
Ah yes, it must have been created when we generated the controller, thanks !